src/Migration/Exception.php

```diff
@@ -4,18 +4,26 @@
 
 class Exception extends \Exception
 {
+    public string $resourceName;
+
     public string $resourceType;
 
     public string $resourceId;
 
-    public function __construct(string $resourceType, string $message, int $code = 0, ?\Throwable $previous = null, string $resourceId = '')
+    public function __construct(string $resourceName, string $resourceType, string $message, int $code = 0, ?\Throwable $previous = null, string $resourceId = '')
```

Comment:
These would probably make more sense as `string $resourceType, string $resourceGroup`, since we pass them in like `Resource::TYPE_TEAM, Transfer::GROUP_AUTH`

Reply:
Yep sounds good :+1: